### PR TITLE
Update dependency and plugin versions

### PR DESCRIPTION
**Pull request by chrishantha**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -826,38 +826,38 @@
         <equinox.osgi.version>3.11.0.v20160603-1336</equinox.osgi.version>
         <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
         <carbon.kernel.version>5.2.0</carbon.kernel.version>
-        <carbon.touchpoint.version>1.1.0</carbon.touchpoint.version>
+        <carbon.touchpoint.version>1.1.1</carbon.touchpoint.version>
         <carbon.kernel.version.range>[5.2.0, 6.0.0)</carbon.kernel.version.range>
-        <carbon.analytics.common.version>6.0.2</carbon.analytics.common.version>
+        <carbon.analytics.common.version>6.0.3</carbon.analytics.common.version>
```
Comment (gimantha) [inline pom.xml]: Shall we bump this to 6.0.4 which is the latest as of now.?

File: pom.xml [hunk 2]
```diff
@@ -826,38 +826,38 @@
...
         <carbon.analytics.common.version.range>[6.0.0, 7.0.0)</carbon.analytics.common.version.range>
-        <carbon.config.version>2.1.2</carbon.config.version>
+        <carbon.config.version>2.1.4</carbon.config.version>
         <carbon.config.version.range>[2.0.0, 3.0.0)</carbon.config.version.range>
```
Comment (gimantha) [inline pom.xml]: This should be [2.1.0, 3.0.0) right?

**Review by gimantha — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (chrishantha) [reply pom.xml]: Yes.. I'll update that.

File: pom.xml (hunk 1 above)
Comment (chrishantha) [reply pom.xml]: Okay. I will do it.

**Review by gimantha — APPROVED** [review]

**Pull request merged**